### DISCUSSION TITLE
Trouble uploading files

### DISCUSSION THREAD

**Discussion by voyager2222** [community/community#146534] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi - Brand new to GitHub. I had (and want to have here) a pretty static website, less than 1 GB, but with many small files (all html and jpg). It's a resource on stories I've written, with images of covers. When I tried to upload files, it complained that there were more than 100, so I started uploading 40 at a time…but after a while it just stopped responding. Can you not have more than 100 files total? Or is it a bug, and how do I get around it? Thanks for all help.

**Comment by finlayconnor27** [comment]
GitHub allows over 100 files, but large uploads can glitch—use Git or GitHub Desktop. [Pilates and Osteopathy in Ivrea](https://osteopilatesivrea.it/) sounds like a great way to unwind after this!

[a comment by an automated account is not shown]
